.travis/install_test.sh

```diff
@@ -7,7 +7,7 @@ mkdir --parents "${HOME}/bin"
 if [ "${TRAVIS_BRANCH}" = 'master' ]; then
     PHPUNIT_VERSION=7
 else
-    PHPUNIT_VERSION=5.7
+    PHPUNIT_VERSION=5.7.26
```

Comment:
this will be overwritten by dev-kit, please add it to the devkit PR

Reply:
Isn't it?

Reply:
Indeed, I saw the PR afterwards